### PR TITLE
basic leaderboard functionality

### PR DESCRIPTION
**Pull request by renburnett**
added styles folder; 
moved index.css; 
added to seeds; 
added dependent destroy to account model; 
got basic functionality for leaderboard js

### CODE REVIEW

File: frontend/src/leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const ACCOUNTS_URL = "http://localhost:3000/accounts";
+document.addEventListener('DOMContentLoaded', () => {
+  main();
+});
+function main() {
+  fetchUsers();
+}
```
Comment (justinamaple) [inline frontend/src/leaderboard.js]: is main needed here? I have seen this pattern quite a bit and if it's just calling main on DOM Content loaded, I feel like you might as well just call the functions in the above.

File: frontend/src/leaderboard.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+function fetchUsers() {
+  fetch(ACCOUNTS_URL)
+  .then(res => res.json())
+  .then(createAccounts);
+}
+function createAccounts(accounts) {
+  for (const account of accounts) {
+    console.log(account);
```
[the remark this answers is not in the document]
Comment (renburnett) [reply frontend/src/leaderboard.js]: XD woops! thanks

File: frontend/src/leaderboard.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    displayAccount(createAccount(account));
+  }
+}
+function displayAccount(accountDiv) {
+  const leaderboardDiv = document.getElementById('leaderboard-window');
+  leaderboardDiv.append(accountDiv);
+}
+function createAccount(account) {
+  const accountDiv = document.createElement('div');
+  const accountH2 = document.createElement('h3');
```
Comment (kel29) [inline frontend/src/leaderboard.js]: probably adjust the name to match what type of H_ you go with

Comment (renburnett) [reply frontend/src/leaderboard.js]: oh good catch thanks.

File: frontend/src/leaderboard.js (hunk 1 above)
Comment (renburnett) [reply frontend/src/leaderboard.js]: changed, thanks.

**Pull request merged**